Sources/CovidCertificateSDK/TrustList/NationalRulesListUpdate.swift

```diff
@@ -23,7 +23,11 @@ class NationalRulesListUpdate: TrustListUpdate {
         let request = CovidCertificateSDK.currentEnvironment.nationalRulesListService.request(reloadIgnoringLocalCache: ignoreLocalCache)
         let (data, response, error) = session.synchronousDataTask(with: request)
 
-        if let httpResponse = response as? HTTPURLResponse, let timeShiftError = detectTimeshift(response: httpResponse) {
+        // Only run timeshift detection if request does not come from cache
```

Comment:
@UBaggeler don't we need to introduce this additional check also in the revocation list and trust list update?

Reply:
Thinking more about it, let's move the timeshift detection only to revocationList request and skip it for the other requests. That way we can use `If-None-Match` and therefore `HTTP 304 Not Modified` for the other requests.